tests/virt/utils.py

```
@@ -399,6 +399,27 @@ def get_non_terminated_pods(client, node):
+    Args:
+        vm: Running VM with SSH access.
+
+    Returns:
+        Sorted list of lspci lines (e.g. ["00:01.0 Display controller: ..."]).
+    """
+    output = run_ssh_commands(
+        host=vm.ssh_exec,
+        commands=["lspci"],
+    )[0].strip()
+    addresses = output.splitlines()
+    LOGGER.info(f"PCI addresses for VM {vm.name}: {addresses}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)